test/jdk/java/awt/Mixing/AWT_Mixing/MixingPanelsResizing.java

```diff
@@ -298,6 +305,19 @@ public static synchronized void fail(String whyFailed) {
         failureMessage = whyFailed;
         mainThread.interrupt();
     }//fail()
+    private static boolean isAlmostEqualColor(Color color, Color refColor) {
+        System.out.println("Comparing color: "+color+" with reference color: "+refColor);
+                if(color.equals(refColor)){
+            return true;
+        } else {
+            return Math.abs(color.getRed() - refColor.getRed()) <
```

Comment:
The indentation of the first `if` is wrong.

Essentially, this `if` condition can be simplified to

```java
return color.equals(refColor)
       || (Math.abs(color.getRed() - refColor.getRed())
              < COLOR_TOLERANCE_MACOSX
           && Math.abs(color.getGreen() - refColor.getGreen())
              < COLOR_TOLERANCE_MACOSX
           && Math.abs(color.getBlue() - refColor.getBlue())
              < COLOR_TOLERANCE_MACOSX;
```
There may be a utility method available for colour tolerance which seems quite common for macOS.